src/api/test/functional/request_controller_test.rb

```diff
@@ -2675,8 +2675,8 @@ def test_auto_accept_request
     # the other one got close because the target does not exist anymore
     get "/request/#{id2}"
     assert_response :success
-    assert_xml_tag(tag: 'state', attributes: { name: 'revoked', when: '2010-07-14T00:00:00', who: 'Iggy' })
-    assert_xml_tag(tag: 'comment', content: 'Permission problem')
+    assert_xml_tag(tag: 'state', attributes: { name: 'declined', when: '2010-07-14T00:00:00', who: 'Iggy' })
+    assert_xml_tag(tag: 'comment', content: "The target package 'TestPack' has been removed")
```

Comment:
@evanrolfe @hennevogel I would need some more time to verify that this is the correct approach. I want to be sure before having this merged.